sci-misc/boinc/files/boinc.init

```diff
@@ -23,30 +23,45 @@ create_work_directory() {
 		chown "${USER}:${GROUP}" "${RUNTIMEDIR}"
 	fi
 
-	if [[ ! -e "${RUNTIMEDIR}"/ca-bundle.crt ]]; then
+	if [ ! -e "${RUNTIMEDIR}"/ca-bundle.crt ]; then
 		ln -s /etc/ssl/certs/ca-certificates.crt "${RUNTIMEDIR}"/ca-bundle.crt
 	fi
 
 	return 0
 }
 
 
+fix_lib_symlinks() {
+	local src="$1"
+	local tgt="$2"
+
+	if test -n "${tgt}"; then
+		# Remove a broken symlink
+		if test -h "${tgt}" \
+			-a "${src}" != "$(readlink "${tgt}")" ; then
+			rm -f "${tgt}"
+		fi
+
+		# symlink the correct path
+		if test -n "${src}" -a -f "${src}" -a ! -h "${tgt}"; then
+			ln -snf "$src" "$tgt"
+		fi
+	fi
+}
+
 cuda_check() {
-	local libtarget="${RUNTIMEDIR}/libcudart.so"
 	local libsource="$(ls -t /opt/cuda/lib*/libcudart.so 2>/dev/null | head -n 1)"
```

Comment:
Actually CUDA isn't multilib, and the `ls` will always find exactly one libcudart.so. I just was afraid it could find two, because of some weird update that went wrong or something like that. 

The library will always be `/opt/cuda/lib64/libcudart.so`, as CUDA does not support non-64bit native development.

Hmm... but wold it really be safe to just hardcode the path?